Scripts/DCS-BIOS/lib/modules/aircraft_modules/Mi-24P.lua

```diff
@@ -1188,4 +1188,9 @@ Mi_24P:define3PosTumb("PLT_SARPP12_MODE", 62, 3001, 357, "SARPP-12", "PILOT SARP
 
 Mi_24P:defineMultipositionSwitch("PLT_JADRO_SQUELCH_SW", 50, 3022, 421, 8, 0.125, "Jadro-1I", "PILOT Jadro-1I Squelch (as Multipos)")
 
+Mi_24P:defineReadWriteRadio("R863_RADIO", 49, 7, 3, 1000, "R-863 Radio")
+Mi_24P:defineReadWriteRadio("JADRO1I_RADIO", 50, 7, 1, 100, "JADRO-1I Radio")
```

Comment:
7 digits and only one decimal place? That would mean it's like 12340.5 - is that right?